### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by cfrandsen09**
Currently most of the work I have done has been in the src/components/LoginPage.js file. This file creates the initial login page people will see when going to apply for a moving truck permit or when they want to check on the status of their application. I have been successful in creating a page that looks mostly how it should. I am having some problems decreasing the size of the input box for the email address. I tried creating a .css file to adjust the width (src/components/LoginPage.css), but seems to be having no impact. I'm not sure if it's a problem with what I've coded or if I need to look closer at the .css file from boston.gov to fix this issue. I am also hoping to get the buttons working to link to other files that we have created, such as src/components/home.js, however, I have only been able to have any success when linking to an already existing url.  
The next step will be to work on the connecting pages and get the functionality of the login page fully working, so that entering in the correct email and password directs a user to their specific application status.

### CODE REVIEW

**Comment by SamuelColacchia** [comment]
You may want to look into project structure. Perhaps separate folders for each component?

Also why is their a login.css and a loginPage.css?

To the issue you where having with email.

`<input type="email" placeholder="Email address" class="txt-f" />` 

You should be able to tag it using the class you gave it. Though assuming you wanted to tag it exclusively you may want to give it a id or give it another class. Say something like this.

`<input type="email" placeholder="Email address" id="some-unique-id" class="txt-f some-new-class" />` 

Then the following css could be used to make it smaller

``` CSS
#some-unique-id {
 width: Xpx;
 height: Xpx;
}
.some-new-class {
 width: Xpx;
 height: Xpx;
}
```

**Comment by DonutPirate** [comment]
Might be worth looking into padding as well, which may help you resize the buttons without making the size static. `<div style={{ padding: '.75em', }}>` is an example from our React app.

**Comment by nicholas-miller** [comment]
I'm having trouble understanding the purpose of all the \<div> tags you're using- most are just used for unused class names, which in turn are not descriptive of their purpose. I've had CSS difficulties due to burying objects in divs.

You can use the \<div> classnames directly in the CSS file- might be a more effective way of changing the spacing than identifying the object by type. It also allows for more control- in the current style, you'd potentially run into trouble if you wanted to add another email input to the page somewhere.

File: README.md [hunk 1]
```diff
@@ -1,31 +1,2434 @@
...
-The New Urban Mechanics will provide access to data, key stakeholders, and other resources as necessary. They will also provide regular feedback (frequency established in partnership with the team).
-
-The prototype delivered by Gordon College Consultants will be used as a proof-of-concept design and catalyst for internal stakeholder conversations around moving day permitting, and may not be adopted as a final solution to the above Issue.
-
-### Moving Truck Permit Dataset - https://data.boston.gov/dataset/open-moving-truck-permits
-### Current Street Sweeping Schedules - https://data.boston.gov/dataset/street-sweeping-schedules
-### Current Moving Permit Reservation System - https://www.boston.gov/how-reserve-parking-spot-your-moving-truck
-### General Moving Day Information - https://www.boston.gov/movingx
-### Proposal- https://docs.google.com/document/d/1U_7Pi0U9DdZ-eEz-trZTXDP3XaLs8rmhaDX_kh44vkM/edit#heading=h.6cxtcn9dvq82
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```
Comment (russtuck) [inline README.md:1]: Did you mean to overwrite your README.md with this generated one?  Having both, with different names, seems logical.

**Pull request closed without merging**